ops/internal/deployer/deployer.go

```diff
@@ -63,7 +63,7 @@ func NewOpDeployer(lgr log.Logger, l1ContractsRelease string, binDir string) (*O
 		l1ContractsRelease: l1ContractsRelease,
 	}
 
-	err := opd.checkBinary(binDir)
+	err := opd.checkBinary(binDir, opDeployerVersion)
```

Comment:
This seems like kind of weird to me. Why can't the l1 contracts release be auto-detected in this case? Why are we passing in the deployer version like this? If we need an override or something it'd be cleaner to extract the deployer bin location out of the opdeployer struct and into some standalone util that gets passed into the struct's constructor.

Reply:
We're trying to cover some edge cases for `import_devnet` to handle like [this](https://github.com/ethereum-optimism/devnets/blob/main/alphanets/interop-rc-alpha/op-deployer/state.json#L14-L15) where the contracts locators are long urls, which we don't currently know how to encode and map to op-deployer versions